maps_dashboards/public/components/tooltip/tooltipTable.tsx

```diff
@@ -153,7 +156,7 @@ const TooltipTable = ({
       </EuiFlexGroup>
       <EuiSpacer size="s" />
       <EuiFlexGroup justifyContent="spaceAround" alignItems="center" gutterSize="none">
-        {showLayerSelection && (
+        {showLayerSelection && options?.length > 1 && (
```

Comment:
Should this be >=1?

Reply:
I think we need pagination only if there are more than 1 layer.